### ISSUE TITLE
Takes quite a lot of CPU

### ISSUE THREAD

**Issue by denieler** [johnlindquist/kit#723]
![Screenshot 2022-02-07 at 12 14 10](https://user-images.githubusercontent.com/2836281/152768528-9f21da37-a5dc-4030-83ad-497a56841f53.png)

To me it takes sometimes a pretty significant amount of CPU 😞 I love this tool, but would even consider stopping using it because of this. I would be happy to contribute fixing this, but need some advises where to look at 🙇‍♂️

**Comment by johnlindquist** [comment]
A video of when you see higher cpu usage would be great. I can't investigate/explain without more information

**Comment by denieler** [comment]
yep, I will try to prepare something. The issue is not persistent 🤷‍♂️

**Comment by johnlindquist** [comment]
Can you describe when you see it happen?

**Comment by denieler** [comment]
btw, thanks for this script 😄 super useful!
